### DISCUSSION TITLE
Welcome to red-tetris-42-project Discussions!

### DISCUSSION THREAD

**Discussion by louchebem06** [louchebem06/red-tetris-42-project#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AudeizReading** [comment]
Merci !

**Reply by louchebem06** [reply]
Like this coment

**Comment by louchebem06** [comment]
oui oui oui

**Comment by AudeizReading** [comment]
Merci
